Description: use alloc for tests so they can work in no-std builds.
Author: Peter Michael Green <plugwash@debian.org>

only in patch2:
--- rust-extprim-1.7.1.orig/src/i128.rs
+++ rust-extprim-1.7.1/src/i128.rs
@@ -2046,2 +2046,3 @@
     use i128::{i128, ZERO, ONE, MIN, MAX};
+    use alloc::vec::Vec;
 
--- rust-extprim-1.7.1.orig/src/lib.rs
+++ rust-extprim-1.7.1/src/lib.rs
@@ -78,2 +78,7 @@
 
+// needed for the tests to run in builds without std
+#[cfg(test)]
+#[macro_use]
+extern crate alloc;
+
 #[macro_use] mod forward;
--- rust-extprim-1.7.1.orig/src/u128.rs
+++ rust-extprim-1.7.1/src/u128.rs
@@ -2046,2 +2046,3 @@
     use u128::{u128, ZERO, ONE, MIN, MAX};
+    use alloc::vec::Vec;
 
